weblate/trans/models/component.py

```diff
@@ -2055,7 +2059,10 @@ def _create_translations(  # noqa: C901
                         "skipping update due to error in parsing template: %s",
                         error.nested,
                     )
-                    self.handle_parse_error(error.nested, filename=self.template)
+                    filelink = self.get_repoweb_link(self.template, "1")
+                    self.handle_parse_error(
+                        error.nested, filename=self.template, filelink=filelink
+                    )
```

Comment:
I don't think we need to parse the filelink all way down as it can be generated when rendering the notification. Somethink like this should do it (in `weblate/accounts/notifications.py`):


```py
class ParseErrorNotification(Notification):
....
    def get_context(
        self, change=None, subscription=None, extracontext=None, changes=None
    ):
        context = super().get_context(change, subscription, extracontext, changes)
        if change:
            context["filelink"] = change.component.get_repoweb_link(change.details.filename, "1")
        return context
```